src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $calendar-width;
+  padding: $padding;
+
+  &__day {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
```

Comment:
Ensure that the SCSS variable `$day-size` is defined elsewhere in your SCSS files. If it is not defined, this will cause a compilation error.